jfx_bridge/bridge.py

```diff
@@ -1128,6 +1128,10 @@ def __get__(self, instance, owner):
         return functools.partial(remote_method, instance)
 
 
+class ConcedeBridgeOverrideException(Exception):
```

Comment:
I like this concept, but prefer a shorter, clearer name. Maybe something like "BridgeOverrideFallback", or "DontUseBridgeOverride". Don't see a need for the Exception suffix - like StopIteration, this exception is not actually an exceptional condition, just a way of doing control flow.

Reply:
I called it `ConcedeOverride` and inherited from `BaseException` "since it is technically not an error." which is a justification used for exceptions like `GeneratorExit`. I feel like the meaning of "Concede" fits really well with the meaning of "override".

This also means an override can have a generic `except Exception:` clause inside it to catch all unintended internal errors which shouldn't be propagated to the caller, unlike intentional Exceptions